packages/codemirror-graphql/src/variables/lint.ts

```diff
@@ -172,7 +172,7 @@ function validateValue(
     Object.keys(type.getFields()).forEach(fieldName => {
       if (!providedFields[fieldName]) {
         const fieldType = type.getFields()[fieldName].type;
-        if (fieldType instanceof GraphQLNonNull) {
+        if (fieldType instanceof GraphQLNonNull && !type.getFields()[fieldName].defaultValue) {
```

Comment:
Now that I see this, we have a lot of calls to `type.getFields()` in the branch of the code that handles input object. Can we store that in a variable somewhere so that we don't need to call it multiple times?

Reply:
of course...
I  did it this way because there was already a bunch of getFileds so I chose the less invasive patch :)